frontend/device/remarkable/device.lua

```diff
@@ -3,9 +3,21 @@ local logger = require("logger")
 
 local function yes() return true end
 local function no() return false end
+local EV_ABS = 3
+local ABS_X = 00
+local ABS_Y = 01
+local ABS_MT_POSITION_X = 53
+local ABS_MT_POSITION_Y = 54
+-- Resolutions from libremarkable src/framebuffer/common.rs
+local screen_width = 1404 -- unscaled_size_check: ignore
+local screen_height = 1872 -- unscaled_size_check: ignore
+local wacom_width = 15725 -- unscaled_size_check: ignore
+local wacom_height = 20967 -- unscaled_size_check: ignore
+local wacom_scale_x = screen_width / wacom_width
+local wacom_scale_y = screen_height / wacom_height
+local TimeVal = require('ui/timeval')
```

Comment:
I'd move this near the very top, between `Generic` & `logger`, to follow the style convention.

Also, in the same vein, double quotes please ;).